pkg/apis/ceph.rook.io/v1/network_test.go

```diff
@@ -43,11 +43,32 @@ func TestNetworkCephSpecLegacy(t *testing.T) {
 }
 
 func TestNetworkCephIsHostLegacy(t *testing.T) {
-	net := NetworkSpec{HostNetwork: true}
+	net := NetworkSpec{ HostNetwork: true}
 
 	assert.True(t, net.IsHost())
 }
 
+func TestNetworkCephIsHost(t *testing.T) {
+	net := NetworkSpec{provider : "host"}
+
+	assert.True(t, net.IsHost())
+	net := NetworkSpec{provider : "host-strict"}
+
+	assert.True(t, net.IsHost())
+}
+
+func TestNetworkCephIsHostStrict(t *testing.T) {
+	net := NetworkSpec{provider : "host-strict"}
+
+	assert.True(t, net.IsHostStrict())
+	net := NetworkSpec{provider : "host"}
+
+	assert.False(t, net.IsHostStrict())
+}
+
+
+
+
```

Comment:
I'm pretty sure `gofmt` (the built-in tool for ensuring go source code has consistent style) will remove double empty lines in go source files.

Reply:
ran `gofmt` and now the blank lines are gone.